itests/org.openhab.binding.mqtt.homie.tests/src/main/java/org/openhab/binding/mqtt/HomieImplementationTest.java

```diff
@@ -310,7 +310,7 @@ public void parseHomieTree() throws InterruptedException, ExecutionException, Ti
         // Publish a retain=false value to MQTT.
         property.getChannelState().publishValue(OnOffType.ON).get();
         // This test is flaky if the MQTT broker does not get a time to "forget" this non-retained value
-        Thread.sleep(50);
+        Thread.sleep(500);
```

Comment:
This will always, even on fast machines, block for half a second.
I opt for either removing this test (although I kind of want this to be tested) or constantly checking every 50 ms in a loop up to half a second. WDYT?

Reply:
done